### ISSUE TITLE
Changing Default Directory of Uploaded Files?

### ISSUE THREAD

**Issue by roomwillow** [projectsend/projectsend#898]
Hello!

Just wanted to start by saying I love this app, super useful and is going to be replacing our previous archaic FTP solution for getting files from clients;

One issue I've come across is getting files to land in our NAS directory, ideally, I would like to be able to change uploads to go to /mnt/*serverName* as opposed to the default /var/www/projectsend/uploads that it does now, alternatively, I *could* change the mount point of the CIFS mount to /var/www/projectsend/uploads... but I'd prefer to keep it in /mnt if possible, is there a configuration option somewhere?

**Comment by snowtr** [comment]
This would make a useful improvement.
I'm starting to deploy this solution and my version will be a virtual Ubuntu server with a NAS mounted for large storage.
This improvement would make my deployment easier to work with.

**Comment by ignacionelson** [comment]
Indeed, a nice improvement and -hope I don't jinx it- a simple one, since it only needs to be a setting.
I'll add this.

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
